experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -379,14 +379,14 @@ Assuming that the first row of the given grading is the grading under Kbar
 
 Weierstrass model over a not fully specified base -- U(1)xU(1) Weierstrass model based on arXiv paper 1507.05954 Eq. (A.1)
 
-julia> associated_literature_models(m)
+julia> birational_literature_models(m)
 1-element Vector{String}:
  "1507_05954-1"
 ```
 """
-function associated_literature_models(m::AbstractFTheoryModel)
-  @req has_associated_literature_models(m) "No associated models known for this model"
-  return get_attribute(m, :associated_literature_models)
+function birational_literature_models(m::AbstractFTheoryModel)
+  @req has_birational_literature_models(m) "No birational models known for this model"
```

Comment:
Maybe "No birationally equivalent models known for this model"?

Reply:
That does sound better